lib/board.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'board_row'
+require_relative '../lib/board_row'
```

Comment:
While we were taking small steps moving the files around, we introduced the relative paths.  As the test still passed, I believe we overlooked removing those paths once they became redundant.  So, for this example, the `board_row.rb` file is in the same relative folder. Therefore you should be able to clean this up further.

```suggestion
require_relative 'board_row'
```

Reply:
This is just one instance, I would suggest checking all the files and cleanup appropriately.

Reply:
Went through every file and fixed.